.agents/personas/CLAUDE.md

```
@@ -138,8 +190,9 @@ Scripts disponiveis no `package.json`:
 - `npm run build`
 - `npm run build:full`
 - `npm run i18n:check`
+- `npm run utf8:check`
 - `npm run perf:budget`
 
 ## Observacao final
 
-Se surgir conflito entre este arquivo e `AI_CONTEXT_INDEX.md`, vale a hierarquia definida no indice canonico. Este arquivo existe para concentrar o contexto mais util para Claude Code sem repetir linguagem promocional ou instrucoes que ja ficaram obsoletas.
+Se surgir conflito entre este arquivo e `AI_CONTEXT_INDEX.md`, vale a hierarquia definida no indice canonico. Este arquivo existe para concentrar o contexto mais util para Claude Code sem criar uma segunda fonte de verdade na raiz do repositorio.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)